package.json

```diff
@@ -41,7 +41,8 @@
     "crypto-js": "3.1.2-2",
     "node-xmpp-client": "^3.0.0",
     "request": "^2.48.0",
-    "strophe.js": "^1.2.8",
+    "strophe.js": "1.2.12",
+    "webrtc-adapter": "^2.1.0",
```

Comment:
So If add webrtc-adapter one more time it will be changed to last connected version.
So I will write about webrtc-adapter as a dependency in release notes.
Is it ok?
@soulfly @IegorKozakov